src/utils/utils.ts

```diff
@@ -0,0 +1,10 @@
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';
+
+const modelToRef = (obj: K8sModel) => `${obj.apiGroup}~${obj.apiVersion}~${obj.kind}`;
+const modelToGroupVersionKind = (obj: K8sModel) => ({
+  version: obj.apiVersion,
+  kind: obj.kind,
+  group: obj.apiGroup,
+});
```

Comment:
It would be good to try to remove these. We don't want plugins to need to do these kinds of transformations.